package.json

```diff
@@ -33,7 +33,7 @@
     "@retailmenot/core-ui-gitignore": "^1.0.1",
     "eslint": "^4.19.1",
     "mock-fs": "^4.5.0",
-    "tap": "^11.1.5"
+    "tap": "^12.0.1"
```

Comment:
You are going to need to update both tap and mock-fs to new versions to make this work.

There is also an outstanding issue with mock-fs that makes it not work past node 10.4.1.  https://github.com/tschaub/mock-fs/issues/238